### ISSUE TITLE
JWT related redesign

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#2582]
Prerequisites: #2530

This is a theme that was deferred when upgrading to Spring Boot 3.x.

Since the mechanism of Spring Security has changed significantly, it is possible to change the JWT design to be more Spring-like. 

> It might be a good idea to follow Spring's new design, although not right away. If anything, it may be better to do this at the same time as adding features such as extending deadlines. In that case, the scope of influence would be unexpectedly large, so it is not a good idea to start working on it right away.

Well, it seems there were other related items as well... 🙄

 - #1831
